thirdparty/loggables/loggables.go

```diff
@@ -9,9 +9,11 @@ package loggables
 import (
 	"net"
 
+	uuid "github.com/satori/go.uuid"
```

Comment:
need to change this to `github.com/ipfs/go-ipfs/Godeps/_workspace/src/github.com/satori/go.uuid`. Gotta make sure we always vendor our dependencies.